flipperdevices/flipperzero-firmware
# FuriThread: Improve state callbacks

# What's new

Split off part of the PR as described in https://github.com/flipperdevices/flipperzero-firmware/pull/3865#issuecomment-2332653925.

# Verification 

* Run a full suite of tests.
* Verify that apps start and stop.
* Verify that RPC via qFlipper still works.

# Checklist (For Reviewer)
